### PR TITLE
Adding rainbow button css

### PR DESCRIPTION
**Pull request by princeo1**
In this pull request I added a button type  css for button which have rainbow animation

Issue:Add animated class to animated buttons

### CODE REVIEW

**Comment by princeo1** [comment]
> In this pull request I added a button type css for button which have rainbow animation
> 
> Issue:Add animated class to animated buttons

Please take care that the following should be the HTML code

<a href="#" class="rainbow-btn" alt="Button"></a>

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @princeo1 thank you for your work. You don't need to create a new file, you need to make changes in `src/components/animated/_rainbow.less` and add an animated class that does what the button does on hover. Please remove the file you added and just make changes in the file specified

File: src/sbuttons.less [hunk 1]
```diff
@@ -6,3 +6,4 @@
 @import "components/_icon.less";
 @import "components/_social.less";
 @import "components/_special.less";
+@import "components/_rainbow.less";
```
Comment (shahednasser) [inline src/sbuttons.less:9]: Please remove this line

**Review by shahednasser — CHANGES_REQUESTED** [review]
You still need to run `npm run compile` and commit the changes

**Comment by shahednasser** [comment]
@princeo1 Please run `npm run compile` and commit the changes

**Comment by shahednasser** [comment]
Hi @princeo1 any update on this?

**Pull request closed without merging**